exporters/otlp/common/src/test/java/io/opentelemetry/exporter/internal/TlsUtilsTest.java

```diff
@@ -0,0 +1,33 @@
+package io.opentelemetry.exporter.internal;
+
+import org.junit.jupiter.api.Test;
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.TrustManager;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class TlsUtilsTest {
```

Comment:
#nit: Can be package private. Also the class is named `TlsUtil`, rather than `TlsUtils`

Reply:
modifier changed. renamed.